passes/source2il.nim

```diff
@@ -77,6 +79,12 @@ proc error(c; message: sink string) =
   ## Sends the error diagnostic `message` to the reporter.
   c.reporter.error(message)
 
+func add[T](bu: var Builder[T], trees: openArray[seq[TreeNode[T]]]) =
```

Comment:
```suggestion
func add(bu; trees: openArray[NodeSeq]) =
```
Not a blocker, but the func doesn't need to be generic, which also allows for a more concise signature.